source/sonata_simulation.rst

```diff
@@ -85,6 +85,7 @@ Parameters defining global simulation settings. As NEURON is the engine used for
    ionchannel_seed                 integer    Optional    A non-negative integer used for seeding stochastic ion channels, default is 0.
    minis_seed                      integer    Optional    A non-negative integer used for seeding the Poisson processes that drive the minis, default is 0.
    synapse_seed                    integer    Optional    A non-negative integer used for seeding stochastic synapses, default is 0.
+   electrodes_file                 text       Optional    Path to the weights file describing the scaling factors for the contributions of each segment's transmembrane current to the LFP/EEG. Format description in sonata_tech.rst
```

Comment:
Perhaps we should use the word `compartment` here (and else where `segments` are used)?